### PR TITLE
TUNE 467: Implement social media buttons visual

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@saeedmb8 @projectmushroom please review 

There should be guideline rules in place:
What kind of images they want to be able to upload?
This should probably be transparent images.

Issues: There should be call to action text 
Link does not get published if media link image is not uploaded?

An example of sharing buttons
screenshot 2017-02-06 19 43 35

**Comment by Mitso** [comment]
Footer social media link 

screenshot 2017-02-07 11 12 08

**Comment by Mitso** [comment]
@saeedmb8 please see updated 

screenshot 2017-02-08 10 15 39

**Comment by saeedmb8** [comment]
@Mitso can we make sure the text in the button is centered if the is no image otherwise👍

**Comment by Mitso** [comment]
@saeedmb8 please see

screenshot 2017-02-08 10 33 21

File: tuneme/templates/new/core/tags/social_media_article.html [hunk 1]
```diff
@@ -1,15 +1,16 @@
-{% load wagtailcore_tags wagtailimages_tags i18n %}
+{% load wagtailcore_tags wagtailimages_tags wagtailsettings_tags i18n %}
+ {% get_settings %}
 
 <!--This currently displays when SocialMedia image is uploaded -->
```
Comment (smarzban) [inline tuneme/templates/new/core/tags/social_media_article.html]: please remove the comment

**Comment by smarzban** [comment]
@Mitso small comment otherwise 👍

**Pull request merged**